content/en/docs/Advanced/configuring_partitions.md

```diff
@@ -89,6 +89,60 @@ Note that there are some caveats in the `extra partitions` setup:
         - mount -o rw /dev/disk/by-label/PARTITION_TWO /opt/extra
 ```
 
+## Manual partitioning
+
+In some cases, it's desired that the user has full control over the partitioning of the disk.
+This can be achieved by setting the `no-format` option to `true` and making sure the disk
+is prepared with the desired partitions before running the installer.
+
+Here is an example config that instructs the installer to use the "second" disk (/dev/vdb)
+and makes sure the disk is prepared with the desired partitions before running the installer:
+
+```yaml
```

Comment:
Example directly copied from the tests: https://github.com/kairos-io/kairos/blob/master/tests/custom_partitioning_test.go